.gitignore

```diff
@@ -62,7 +62,6 @@ env.bak/
 venv.bak/
 
 .settings/
-model_cache/
```

Comment:
Leave this here as otherwise it will create a lot of clutter

Reply:
yes, sorry, I removed that for pushing the last model